docs/src/Button.doc.js

```diff
@@ -127,7 +127,7 @@ card(
   <Example
     name="Example"
     defaultCode={`
-<Button text="Medium sized button" inline />
+<Button href="http://www.pinterest.com" text="Medium sized button" target='blank' type="link" inline />
```

Comment:
@AlbertCarreras let's not make this the default example and add a new one instead.

Reply:
Oh I know, I did this because it's how I'm testing while developing